Crashlytics/Crashlytics/Models/Record/FIRCLSReportAdapter.m

```diff
@@ -193,6 +193,9 @@ - (google_crashlytics_FilesPayload)protoFilesPayload {
     if ([filename.pathExtension.lowercaseString isEqualToString:@"clsrecord"]) {
       [clsRecords addObject:[self.folderPath stringByAppendingPathComponent:filename]];
     }
+    if ([filename.pathExtension.lowercaseString isEqualToString:@"symbolicated"]) {
```

Comment:
Add an OR clause to the if statement above?

Reply:
Done